### PR TITLE
Fixed docker and vagrant provisioning

### PR DESCRIPTION
**Pull request by mkurek**
- fixed createsuperuser script (use RalphUser now)
- added ldap lib to provisioning
- use Python3 (only) everywhere

### CODE REVIEW

**Comment by ghost** [comment]
:ok:

File: docker/createsuperuser.py [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
-print('Updating superuser info')
 import os
+import django
+
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "ralph.settings")
-from django.contrib.auth.models import User
-u = User.objects.get(username='ralph')
+django.setup()
+
+from ralph.accounts.models import RalphUser
```
Comment (szok) [inline docker/createsuperuser.py]: Use get_user_model()

File: vagrant/provisioning_scripts/init_mysql.sh [hunk 2]
```diff
@@ -4,5 +4,9 @@ echo "CREATE DATABASE ralph_ng DEFAULT CHARACTER SET 'utf8'" | mysql -u root
 echo "GRANT ALL ON ralph_ng.* TO ralph_ng@'%' IDENTIFIED BY 'ralph_ng'; FLUSH PRIVILEGES" | mysql -u root
 
 /home/vagrant/bin/ralph migrate
+
 /home/vagrant/bin/ralph createsuperuser --noinput --username ralph --email ralph@allegrogroup.com
 /home/vagrant/bin/python vagrant/provisioning_scripts/createsuperuser.py
+
+/home/vagrant/bin/ralph sitetreeload
```
Comment (szok) [inline vagrant/provisioning_scripts/init_mysql.sh]: maybe change to 'make menu'

**Comment by szok** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**